DatabaseMySQL/Dump20221018.sql

```diff
@@ -0,0 +1,200 @@
+-- MySQL Script generated by MySQL Workbench
+-- Tue Oct 18 09:17:06 2022
+-- Model: New Model    Version: 1.0
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION';
+
+-- -----------------------------------------------------
+-- Schema dbQuiz
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema dbQuiz
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `dbQuiz` DEFAULT CHARACTER SET utf8 ;
+USE `dbQuiz` ;
+
+-- -----------------------------------------------------
+-- Table `dbQuiz`.`users`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `dbQuiz`.`users` (
+  `user_id` BIGINT(20) NOT NULL,
+  `full_name` VARCHAR(50) NOT NULL,
+  `date_of_birth` DATE NOT NULL DEFAULT 'dd/mm/yyyy',
```

Comment:
set default z báo lỗi á em
<img width="788" alt="image" src="https://user-images.githubusercontent.com/66414511/196322585-ac8de683-3264-4b3c-9162-1cd7c7f6a28f.png">

Reply:
để mặc định là DATE NOT NULL được rồi em. Trong sql nó sẽ lưu theo dạng yyyy-mm-dd